apps/admin/frontend/src/components/import_cvrfiles_modal.tsx

```diff
@@ -148,10 +148,9 @@ export function ImportCvrFilesModal({ onClose }: Props): JSX.Element | null {
           <Prose>
             <h1>Error</h1>
             <p>
-              There was an error reading the content of the file{' '}
+              There was an error reading the contents of{' '}
               <strong>{currentState.filename}</strong>:{' '}
-              {currentState.errorMessage}. Please ensure this file only contains
-              valid CVR data for this election.
+              {currentState.errorMessage}
```

Comment:
I noticed some wonkiness with the current CVR import error messaging, e.g. we mention reading a file but we now read a directory. We also had some double periods `..` because nearly all of the server-returned user-friendly error messages include a period already. I also decided to remove the hardcoded "Please ensure this file only contains valid CVR data for this election" help message because 1) we export a directory not a file, and 2) all the server-returned error messages include more specific guidance that this second generic message distracts from.

Note that I haven't yet replaced all instances of the variable name `filename` with, say, `directoryName` because the long-term state of CVR exports/imports is still in a bit of flux.

Reply:
Nice catch. I think your cleanup makes sense.

This is one example of why I think it makes more sense for the backend to return error identifiers rather than full user-facing error copy. (I see you've been using that approach in this PR.)